WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -46,7 +46,7 @@ struct PointOfSaleDashboardView: View {
         .animation(.easeInOut, value: viewModel.isInitialLoading)
         .background(Color.posBackgroundGreyi3)
         .navigationBarBackButtonHidden(true)
-        .posModal(isPresented: $totalsViewModel.showsCardReaderSheet, content: {
+        .posModal(isPresented: $totalsViewModel.showsCardReaderSheet, fixedHeight: false, content: {
```

Comment:
```suggestion
        .posModal(isPresented: $totalsViewModel.showsCardReaderSheet, content: {
```

Height should be fixed – otherwise the card reader connection modal changes size as the connection progresses, which is quite distracting.